index.html

```diff
@@ -1,57 +1,69 @@
-      <li>2 tsp baking soda</li>
-      <li>1/2 tsp salt</li>
-      <li>4 cups chocolate chips</li>
-      <li>1 cup chopped walnuts or pecans(optional)</li>
-    </ul>
-  </section>
-
-  <section id="instructions">
-    <h2>Instructions</h2>
-    <ol class="list">
-      <li>Preheat oven to 350 degrees Beat butter and sugars together until light and fluffy.</li>
-      <li>Stir in pudding mix milk and vanilla extract.</li>
-      <li>Beat in eggs.</li>
-      <li>Add dry ingredients and stir until combined.</li>
-      <li>Stir in chocolate chips and nuts.</li>
-      <li>Place 1 1/2 inch balls of dough 2 inches apart on an un greased cookie sheet.</li>
-      <li>Bake 8-10 minutes or until golden brown.</li>
-    </ol>
-  </section>
-
-  <section id="nutrition">
-    <h2>Nutrition Information</h2>
-    <p>Probably bad for you!</p>
-  </section>
-
-  <footer>
-    <p>Recipe is inspired by <a href="http://www.opensourcefood.com/people/Amanori/recipes/mamas-recipe-the-best-chocolate-chip-cookies">Open Source Food</a></p>
-  </footer>
-
-</body>
+  <head>
+    <meta name="description" content="Demo project using HTML, CSS, & JavaScript">
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <title>Grandma's Cookies, yo</title>
+  </head>
+
+  <body>
+
+    <header>
+      <h1>The Best Chocolate Chip Cookies</h1>
+      <p>Recipe by: Grandma <br> Prep Time: 45 Min</p>
+      <img src="./img/cookies.jpg">
+    </header>
+
+    <section id="ingredients">
+      <h2>INGREDIENTS</h2>
+      <ul>
+        <li>1 1/2 cup (3 sticks) softened butter</li>
+        <li>1 cup brown sugar</li>
+        <li>1 cup granulated sugar</li>
+        <li>1 Tbl vanilla instant pudding powder</li>
+        <li>2 Tbl milk </li>
+        <li>2 Tbl vanilla extract</li>
+        <li>2 eggs</li>
+        <li>4 cups all purpose flour</li>
+        <li>2 tsp baking soda</li>
+        <li>1/2 tsp salt</li>
+        <li>4 cups chocolate chips</li>
+        <li>1 cup chopped walnuts or pecans(optional)</li>
+      </ul>
+    </section>
+
+    <section id="instructions">
+      <h2>Instructions</h2>
+      <ol class="list">
+        <li>Preheat oven to 350 degrees Beat butter and sugars together until light and fluffy.</li>
+        <li>Stir in pudding mix milk and vanilla extract.</li>
+        <li>Beat in eggs.</li>
+        <li>Add dry ingredients and stir until combined.</li>
+        <li>Stir in chocolate chips and nuts.</li>
+        <li>Place 1 1/2 inch balls of dough 2 inches apart on an un greased cookie sheet.</li>
+        <li>Bake 8-10 minutes or until golden brown.</li>
+      </ol>
+    </section>
+
+    <section id="nutrition">
+      <h2>Nutrition Information</h2>
+      <p>Bad for you, but good for the soul!</p>
+    </section>
+
+    <footer>
+      <p>Recipe is inspired by <a href="http://www.opensourcefood.com/people/Amanori/recipes/mamas-recipe-the-best-chocolate-chip-cookies" target=blank>Open Source Food</a></p>
```

Comment:
Looks like the html was smart enough to figure it out, but usually, you want `target=_blank` here (note the underscore)